scripts/minikube_deploy.py

```diff
@@ -64,49 +72,89 @@ def main():
+            chartpress_dir = os.path.join(
+                dependency_dir(dep['repo_name']), dep['chartpress_dir']
+            )
             chart_name = dep.get('chart_name', dep['repo_name'])
 
-            with open(os.path.join(chartpress_dir, chart_name, 'Chart.yaml'), 'rt') as f:
+            with open(
+                os.path.join(chartpress_dir, chart_name, 'Chart.yaml'), 'rt'
+            ) as f:
                 chart = yaml.load(f)
             version = chart.get('version')
 
-            req = next(filter(lambda x: x.get('name') == chart_name, renku_requirements.get('dependencies')))
+            req = next(
+                filter(
+                    lambda x: x.get('name') == chart_name,
+                    renku_requirements.get('dependencies')
+                )
+            )
             req['version'] = version
-            req['repository'] = 'file://{}'.format(os.path.abspath(os.path.join(chartpress_dir, chart_name)))
+            req['repository'] = 'file://{}'.format(
+                os.path.abspath(os.path.join(chartpress_dir, chart_name))
+            )
 
         with open(os.path.join(tmp, 'renku', 'requirements.yaml'), 'wt') as f:
             yaml.dump(renku_requirements, f)
 
         run(['cat', 'renku/requirements.yaml'], cwd=tmp).check_returncode()
 
         package_chart('renku', tmp, tmp)
-        renku_chart = os.path.abspath(glob(os.path.join(tmp, 'renku-*.tgz'))[0])
+        renku_chart = os.path.abspath(
+            glob(os.path.join(tmp, 'renku-*.tgz'))[0]
+        )
 
         helm_deploy_cmd = [
-            'helm', 'upgrade', release,
+            'helm',
+            'upgrade',
+            release,
             renku_chart,
             '--install',
-            '--namespace', namespace,
-            '-f', os.path.join(renku_chartpress_dir, 'minikube-values.yaml'),
-            '--set', 'global.renku.domain={mip}'.format(mip=minikube_ip()),
-            '--set', 'ui.gitlabUrl=http://{mip}/gitlab'.format(mip=minikube_ip()),
-            '--set', 'ui.jupyterhubUrl=http://{mip}/jupyterhub'.format(mip=minikube_ip()),
-            '--set', 'ui.gatewayUrl=http://{mip}/api'.format(mip=minikube_ip()),
-            '--set', 'gateway.keycloakUrl=http://{mip}'.format(mip=minikube_ip()),
-            '--set', 'gateway.gitlabUrl=http://{mip}/gitlab'.format(mip=minikube_ip()),
-            '--set', 'notebooks.jupyterhub.hub.extraEnv.GITLAB_URL=http://{mip}/gitlab'.format(mip=minikube_ip()),
-            '--set', 'notebooks.jupyterhub.hub.services.gateway.oauth_redirect_uri=http://{mip}/api/auth/jupyterhub/token'.format(mip=minikube_ip()),
-            '--set', 'notebooks.gitlab.registry.host=10.100.123.45:8105',
-            '--set', 'gitlab.registry.externalUrl=http://10.100.123.45:8105/',
-            '--timeout', '1800',
+            '--namespace',
+            namespace,
+            '-f',
+            os.path.join(renku_chartpress_dir, 'minikube-values.yaml'),
+            '--set',
```

Comment:
if we're expanding this, let's use `set-string`

Reply:
I didn't realize that was an option -- fixed in c7d0cf2